workflow/scripts/check_srf.py

```diff
@@ -24,6 +24,12 @@ def check_srf(
         Path, typer.Argument(help="The path to the realisation for the SRF.")
     ],
     srf_ffp: Annotated[Path, typer.Argument(help="The path to the SRF file to check.")],
+    velocity_model_path: Annotated[
```

Comment:
Just a potential rename of registry_velocity_model_path to remove some potential confusion. Along with all the other ones